database/insertData.js

```diff
@@ -6,20 +6,18 @@ const graphQLendpoint = config.graphQLendpoint
 
 const queries = [
   // Template A -- Test - General Registration (Feature showcase)
-  `mutation {
+  `
+
+  mutation {
     createTemplate(
       input: {
         template: {
           code: "TestRego"
           name: "Test -- General Registration"
           isLinear: false
-          startTitle: "This is the general registration for feature showcase"
-          startMessage: 
-            """Proof of identity (Passport, Drivers license)\n
-            Proof of your medical certification\n
-            Drug ingredient list\n
-            Product images\n
-            Packging images\n"""
+          startMessage: """
+          ## This is the general registration for feature showcase\n- Proof of identity (Passport, Drivers license)\n- Proof of your medical certification\n- Drug ingredient list\n- Product images\n- Packging images\n
+          """
```

Comment:
The `##` needs to start right after the triple-quote, otherwise it's adding additional whitespace at the start of the string.

Reply:
Also, no need for final line break `\n`.

Reply:
This doesn't work 😞  With """ running the `Prettify` is changing it back to have the first empty line. Not sure how to fix this. Any ideas?

Reply:
Done: [11dd250](https://github.com/openmsupply/application-manager-server/pull/162/commits/11dd250da726244246bd35d52cb5478c78d88665)